main.c

```diff
@@ -524,6 +528,139 @@ init_vt(struct vkcube *vc)
    return 0;
 }
 
+static int
+create_drm_image(struct vkcube *vc, struct vkcube_buffer *b,
+    uint64_t drm_format, uint64_t drm_format_mod)
+{
+      int fd;
+
+      b->gbm_bo = gbm_bo_create_with_modifiers(vc->gbm_device, vc->width, vc->height,
+                                drm_format, &drm_format_mod, 1);
```

Comment:
Should we use `gbm_bo_create_with_modifiers2()` so that we can set `GBM_BO_USE_RENDERING`?

Reply:
ack